.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
- [ ] Perfect! You added `test.md`, bit you can see this file still in your repository. I think you did not delete it first, before pushing changes. Please, first delete `test.md` file from your repository and then push your changes.